### PR TITLE
prepare next release

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

**Review by slashben — APPROVED** [review]
I am looking only at the KDR part. It looks OK!

File: charts/kubescape-operator/values.yaml [hunk 1]
```diff
@@ -559,29 +565,35 @@ nodeAgent:
 clamav:
   name: clamav
   image:
-    repository: quay.io/armosec/klamav
-    tag: beta5
-    pullPolicy: Always
+    # -- source code: https://github.com/kubescape/node-agent/tree/main/clamav
+    repository: quay.io/kubescape/klamav
+    tag: 1.2.0-6_base
```
Comment (dwertent) [inline charts/kubescape-operator/values.yaml:570]:
```suggestion
    tag: v1.2.0
```

File: charts/kubescape-operator/values.yaml [hunk 2]
```diff
@@ -559,29 +565,35 @@ nodeAgent:
 clamav:
   name: clamav
   image:
-    repository: quay.io/armosec/klamav
-    tag: beta5
-    pullPolicy: Always
+    # -- source code: https://github.com/kubescape/node-agent/tree/main/clamav
+    repository: quay.io/kubescape/klamav
```
Comment (dwertent) [inline charts/kubescape-operator/values.yaml:569]: `klamav` seriously? 🤓

File: charts/kubescape-operator/values.yaml [hunk 3]
```diff
@@ -78,8 +78,9 @@ capabilities:
   #
   runtimeObservability: enable
   networkPolicyService: enable
-  runtimeDetection: disable
-  malwareDetection: disable
+  runtimeDetection: enable
```
Comment (dwertent) [inline charts/kubescape-operator/values.yaml:81]:
```suggestion
  runtimeDetection: disable
```

File: charts/kubescape-operator/values.yaml [hunk 4]
```diff
@@ -78,8 +78,9 @@ capabilities:
...
+  malwareDetection: enable
```
Comment (dwertent) [inline charts/kubescape-operator/values.yaml:82]:
```suggestion
  malwareDetection: disable
```

File: charts/kubescape-operator/values.yaml [hunk 5]
```diff
@@ -103,8 +104,8 @@ configurations:
 
 # installation of the alertCRD chart
 alertCRD:
-  installDefault: false # install the default CRD
-  scopeClustered: false # it is better to have the CRDs in the cluster scope
+  installDefault: true # install the default CRD
```
Comment (dwertent) [inline charts/kubescape-operator/values.yaml:107]:
```suggestion
  installDefault: false # install the default CRD
```

File: charts/kubescape-operator/values.yaml [hunk 6]
```diff
@@ -103,8 +104,8 @@ configurations:
...
+  scopeClustered: true # it is better to have the CRDs in the cluster scope
```
Comment (dwertent) [inline charts/kubescape-operator/values.yaml:108]:
```suggestion
  scopeClustered: false # it is better to have the CRDs in the cluster scope
```

**Review by dwertent — CHANGES_REQUESTED** [review]

**Review by dwertent — APPROVED** [review]

**Pull request merged**